vms/volunteer/views.py

```diff
@@ -1,242 +1,242 @@
-                'report_list': report_list,
-                'total_hours': total_hours,
-                'notification': True,
-                'job_list': job_list,
-                'event_list': event_list,
-                'selected_event': event_name,
-                'selected_job': job_name
-            })
-
-
-@login_required
-@admin_required
-def search(request):
-    if request.method == 'POST':
-        form = SearchVolunteerForm(request.POST)
-        if form.is_valid():
-
-            first_name = form.cleaned_data['first_name']
-            last_name = form.cleaned_data['last_name']
-            city = form.cleaned_data['city']
-            state = form.cleaned_data['state']
-            country = form.cleaned_data['country']
-            organization = form.cleaned_data['organization']
-
-            search_result_list = search_volunteers(
-                first_name, last_name, city, state, country, organization)
-            return render(
-                request, 'volunteer/search.html', {
-                    'form': form,
-                    'has_searched': True,
-                    'search_result_list': search_result_list
-                })
-    else:
-        organizations_list = get_organizations_ordered_by_name()
-        form = SearchVolunteerForm()
-
-    return render(
-        request, 'volunteer/search.html', {
-            'organizations_list': organizations_list,
-            'form': form,
-            'has_searched': False
-        })
+# standard library
+import os
+
+# third party
+from braces.views import LoginRequiredMixin
+
+# Django
+from django.conf import settings
+# from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required
+from django.core.servers.basehttp import FileWrapper
+from django.core.urlresolvers import reverse, reverse_lazy
+from django.http import Http404, HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+from django.views.generic.detail import DetailView
+from django.views.generic import ListView, View
+from django.views.generic.edit import FormView, UpdateView
+from django.utils.decorators import method_decorator
+
+# local Django
+from administrator.utils import admin_required
+from event.services import get_signed_up_events_for_volunteer
+from job.services import get_signed_up_jobs_for_volunteer
+from organization.services import get_organization_by_id, get_organizations_ordered_by_name
+from shift.services import get_volunteer_report, calculate_total_report_hours
+from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
+from volunteer.models import Volunteer
+from volunteer.services import get_volunteer_by_id, delete_volunteer_resume,search_volunteers, get_volunteer_resume_file_url, has_resume_file
+from volunteer.validation import validate_file
+from volunteer.utils import vol_id_check
+from vms.utils import check_correct_volunteer
+
+
+@login_required
```

Comment:
Why do you have all this unrelated changes here ? 

This should just be a 2 line PR and nothing else.

Reply:
Fixed it. Please review.